.changeset/sixty-camels-design.md

```diff
@@ -0,0 +1,7 @@
+---
+'@graphql-hive/nestjs': patch
```

Comment:
(I am not a fan of v0 versioning)

This package is far to simple to start with v0... Using v0 packages feels unstable and forever breaking.

I'd recommend releasing a major and start with v1, therefore showing our confidence behind the release and creating a good incentive for users to try it out.

In general I think we're too much invested in zero versionining and some packages get stuck there forever. IMHO, everything we test and intend to ship, with a smaller API not prone to changes - should always start with v1.

```suggestion
'@graphql-hive/nestjs': major
```